ydb/core/blobstorage/pdisk/blobstorage_pdisk_ut.cpp

```diff
@@ -1488,13 +1499,19 @@ Y_UNIT_TEST_SUITE(TPDiskTest) {
         ui32 logBuffSize = 250;
         ui32 chunkBuffSize = 128_KB;
 
-        for (ui32 testCase = 0; testCase < 8; testCase++) {
+        for (ui32 testCase = 0; testCase < 16; testCase++) {
             Cerr << "restart# " << bool(testCase & 4) << " start with noop scheduler# " << bool(testCase & 1)
-                << " end with noop scheduler# " << bool(testCase & 2) << Endl;
+                << " end with noop scheduler# " << bool(testCase & 2) << " multiple encryption threads# "
+                << bool(testCase & 8)<< Endl;
             testCtx.SafeRunOnPDisk([=] (NPDisk::TPDisk* pdisk) {
                 pdisk->UseNoopSchedulerHDD = testCase & 1;
                 pdisk->UseNoopSchedulerSSD = testCase & 1;
             });
+            if (testCase & 8) {
```

Comment:
I don't like this personally, but seeing that it's been like this already in this test, I can't ask you to change it